docs/addons/konveyor.md

```diff
@@ -0,0 +1,17 @@
+# Konveyor Amazon EKS Blueprints AddOn
+
+This add-on installs [Konveyor](https://konveyor.github.io/).
```

Comment:
Please elaborate on the use cases  answering the question of why the customer would use this addon, expanding on scenarios where this product is applicable.

Reply:
done

Reply:
Should this point to a different doc portal for tackle2 operator? The documentation page here is misleading and calls out the[ unmaintained repo](https://github.com/konveyor/konveyor.github.io/blob/main/content/_index.md)  as the source which is confusing (to me and I assume the target audience).

Reply:
https://konveyor.github.io/ is good to use for Konveyor documentation.   The repo was stale and needed attention, hence we updated it ~2 weeks ago and are working to revamp this as the central documentation for Konveyor going forward.